install.sh

```diff
@@ -113,19 +143,22 @@ if [ -d "${CREW_PREFIX}" ]; then
     echo_info "Continue?"
     select continue in "Yes" "No"; do
       if [[ "${continue}" == "Yes" ]]; then
-        sudo find "${CREW_PREFIX}" -mindepth 1 -delete
+        # Let's see if avoiding sudo breaks anything.
+        # sudo find "${CREW_PREFIX}" -mindepth 1 -delete
+        find "${CREW_PREFIX}" -mindepth 1 -delete
         break 2
       else
         exit 1
       fi
     done
   fi
 else
-  sudo mkdir "${CREW_PREFIX}"
+  echo_error "${CREW_PREFIX} does not exist. (It should be mounted!)"
```

Comment:
Good point.  Maybe say `It should be created or mounted prior to install`?

Reply:
done.